docs: offline installation improvements (#632)
Volue-Public/energy-mesh-python: docs/source/installation.rst

Setup for users (offline environments)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To install the Mesh Python SDK on a machine without internet connection, you first need to
prepare the required wheel packages on a machine with internet connection and then copy them to the
target machine.

.. important::
    The machine used to prepare the wheels **must have the same Python version and
    platform** (e.g. Windows/Linux, 64-bit) as the target offline machine. Wheel files
    contain platform-specific tags (e.g. ``cp313-cp313-win_amd64``) and are not
    interchangeable between Python versions or platforms.

**Step 1 — Build wheels (on a machine with internet access)**

Build the Mesh Python SDK wheel together with all of its runtime dependencies::

>     # Replace X.Y.Z by the appropriate volue.mesh package version

Reviewer: My mistake, I always confuse "replace by" with "replace with".

```suggestion
    # Replace X.Y.Z with the appropriate volue.mesh package version
```

Setup for users (offline environments)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> To install Mesh Python SDK on a computer without internet connection you first need to

Reviewer: ```suggestion
To install the Mesh Python SDK on a machine without internet connection, you first need to
```

> prepare wheel packages on a computer with internet connection and then copy them to the

Reviewer: ```suggestion
prepare the required wheel packages on a machine with internet connection and then copy them to the
```

target machine.

.. important::

>     The computer used to prepare the wheels **must have the same Python version and

Reviewer: ```suggestion
    The machine used to prepare the wheels **must have the same Python version and
```

    platform** (e.g. Windows/Linux, 64-bit) as the target offline machine. Wheel files
    contain platform-specific tags (e.g. ``cp313-cp313-win_amd64``) and are not
    interchangeable between Python versions or platforms.

> **Step 1 — Build wheels (on a computer with internet access)**

Author: ```suggestion
**Step 1 — Build wheels (on a machine with internet access)**
```

Build the Mesh Python SDK wheel together with all of its runtime dependencies::

>     # Use correct volue.mesh package version.

Reviewer: ```suggestion
    # Replace X.Y.Z by the appropriate volue.mesh package version
```

    python -m pip wheel git+https://github.com/Volue-Public/energy-mesh-python@vX.Y.Z -w ./offline_installer

This will:

- Build the ``volue.mesh`` package into a ``.whl`` file.
- Download pre-built wheels for all runtime dependencies into the same directory.

.. note::
    See :doc:`versions` if you need a specific Mesh version.

**Step 2 — Copy to the target machine**

Copy the entire ``offline_installer`` directory to the target computer without internet

> connection.

Reviewer: ```suggestion
Copy the entire `offline_installer` directory to the target machine.
```

**Step 3 — Install on the target machine**

> In the directory with the downloaded wheels execute::

Reviewer: ```suggestion
Finally, run the following on the directory with the copied wheels::
```
